pkgs/applications/misc/monitor/default.nix

```
@@ -0,0 +1,66 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, ninja
+, vala
+, pkgconfig
+, pantheon
+, python3
+, gettext
+, glib
+, gtk3
+, bamf
+, libwnck3
+, libgee
+, libgtop
+, libxml2
+, wrapGAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "monitor";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "stsdc";
+    repo = "monitor";
+    rev = version;
+    sha256 = "17z1m193s7qygavfwd8qsw97blxbfmq9gnsymdjlc1ddk8hldw0z";
+  };
+
+  nativeBuildInputs = [
+    gettext
+    meson
+    ninja
+    vala
+    pkgconfig
```

```suggestion
    pkg-config
```

This is an alias yet to be deprecated

---

Good to know, thank's